serverless/functions/join-stream-as-speaker.js

```diff
@@ -207,7 +223,9 @@ module.exports.handler = async (context, event, callback) => {
   response.setBody({
     token: token.toJwt(),
     sync_object_names: {
+      viewers_map: 'viewers',
       raised_hands_map: `raised_hands`,
+      viewer_document: `viewer-${user_identity}`,
```

Comment:
I know we use this convention right now, but is the `viewer-` prefix required?

Reply:
We could technically get away without the prefix for this file now but it may get us in the future if we need another user specific sync object. And I think it's easier to follow our sync design if all objects have a descriptive name. It will be easier to refer to in documentation I think. So I think it's a good convention.